pallets/automation-time/src/lib.rs

```diff
@@ -568,9 +616,15 @@ pub mod pallet {
 			last_time_slot: u64,
 		) -> (Weight, u64) {
 			if current_time_slot != last_time_slot {
-				let mut missed_tasks = Self::get_task_queue();
+				let missed_tasks = Self::get_task_queue();
 				let mut missed_queue = Self::get_missed_queue();
-				missed_queue.append(&mut missed_tasks);
+				for missed_task in missed_tasks {
```

Comment:
We will need a way to do this without having to touch every item here by hand. We need thus to bulk moves